draft.tex

```diff
@@ -742,7 +742,14 @@ \section*{Outlook}
 aims to verify the correctness of scientific software. As our
 experience has shown, it is far easier to graft publication onto a
 software development platform than to integrate software reviewing
-into a publishing platform.
+into a publishing platform. Furthermore, tools that will allow for the automated
+validation of computational models and the automated verification of correctness are being actively 
+developed in the community (see e.g. 
+\href{https://github.com/scidash/sciunit}{SciUnit} or 
+\href{https://github.com/OpneSourceBrain/osb-model-validation}{OSB-model-validation}). 
```

Comment:
There's a typo in the link; I guess the correct link is https://github.com/OpenSourceBrain/osb-model-validation.